### PR TITLE
Simplify dashboard

### PR DESCRIPTION
**Pull request by melinath**
An attempt at simplifying the dashboard. This would obviate #816. Changes are:

* Removes register/website/facebook/manage buttons from dashboard.
* Moves registration status onto event banner image (where manage button was)
* Makes banner image slightly taller so the whole thing actually displays
* Tightens margins in event metadata (name, date, location)
* Removes date card to cover up less of the banner image
* Removes verbose alerts below event image (since users can be expected to view a specific order to find those details)
* Linkifies event name.

screen shot 2017-02-10 at 2 58 58 pm

See previous discussion in #834

### CODE REVIEW

**Comment by nmorduch** [comment]
This is good, though I would prefer not to lose the manage button on the homepage. Maybe we can have both of them?

**Comment by nmorduch** [comment]
Just changed to this:

image

Also, I was fond of those little date cards, but don't really care enough to argue about it.

File: brambling/static/brambling/sass/modules/_event-listing.sass [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .event-listing-image
 	width: 100%
-	padding-top: 27%
+	padding-top: (300 / 940) * 100%
```
Comment (emilyhorsman) [inline brambling/static/brambling/sass/modules/_event-listing.sass]: Can we document where this calculation comes from? Also sass has a [`percentage`](http://sass-lang.com/documentation/Sass/Script/Functions.html#percentage-instance_method) function for what I think you're trying to do with the `* 100%` deal. :)

File: brambling/static/brambling/sass/modules/_event-listing.sass [hunk 2]
```diff
@@ -9,32 +9,16 @@
...
-	line-height: 1em
-	font-size: floor(($font-size-base * 1.5))
-
 .event-listing
 	& > .relative-right
 		right: 6px
 	& > .relative-top
 		top: 6px
+
+	& > h3
```
Comment (emilyhorsman) [inline brambling/static/brambling/sass/modules/_event-listing.sass]: The ampersand here is unnecessary. Do we have a style rule for being explicit about this?

File: brambling/static/brambling/sass/modules/_event-listing.sass [hunk 3]
```diff
@@ -9,32 +9,16 @@
...
+		margin-top: 10px
+		margin-bottom: 5px
+
+	& > h5
```
Comment (emilyhorsman) [inline brambling/static/brambling/sass/modules/_event-listing.sass]: (See prev comment about ampersand above)

File: brambling/static/brambling/sass/modules/_text.sass [hunk 4]
```diff
@@ -70,6 +70,10 @@ h1, .h1
 
 //// Buttons
 
+// There was a weird line height thing going on with buttons.
```
Comment (emilyhorsman) [inline brambling/static/brambling/sass/modules/_text.sass]: Can you be more specific in documenting what this `line-height` declaration accomplishes?

File: brambling/templates/brambling/_event_list_item.html [hunk 5]
```diff
@@ -1,94 +1,71 @@
...
+		{% if is_registered %}
+			{% if event.order and event.order.has_cart %}
+				<a href="{{ event.get_absolute_url }}" class="btn btn-default tipped" title="Complete your registration now!" data-placement="bottom">
+					<i class="fa fa-exclamation-circle"></i> Cart expires in
+
+					{% td_to_dict event.order.cart_expire_time|td_timeuntil as time_left %}
+
+					<strong data-countdown="timer">
+						{% if time_left.days %}<span data-countdown="days">{{ time_left.days }}</span> day{{ time_left.days|pluralize }}{% endif %}
+						{% if time_left.hours %}<span data-countdown="hours">{{ time_left.hours }}</span>:{% endif %}<span data-countdown="minutes">{% if time_left.minutes < 10 and time_left.hours %}0{% endif %}{{ time_left.minutes }}</span>:<span data-countdown="seconds">{% if time_left.seconds < 10 %}0{% endif %}{{ time_left.seconds }}</span>
```
Comment (emilyhorsman) [inline brambling/templates/brambling/_event_list_item.html]: Can we clean all this up a bit? It being on one smushed line makes this hard to parse and would make future amendments difficult.

File: brambling/static/brambling/sass/modules/_event-listing.sass (hunk 2 above)
Comment (melinath) [inline brambling/static/brambling/sass/modules/_event-listing.sass]: I'm not gonna change this because it matches the rest of the file and I don't think there's a compelling reason _not_ to do this...

File: brambling/static/brambling/sass/modules/_text.sass (hunk 4 above)
Comment (melinath) [inline brambling/static/brambling/sass/modules/_text.sass]: AFAICT this just increases the line height of buttons slightly, but this was a @nmorduch change - pinging in case she wants to add anything.

File: brambling/templates/brambling/_event_list_item.html (hunk 5 above)
Comment (melinath) [inline brambling/templates/brambling/_event_list_item.html]: It looks like it's probably crammed on one line because we can't allow whitespace between the things on this line.
